### PR TITLE
Undisposals the warden's office

### LINKED ISSUE #894: Heliostation's warden office has a useless disposal bin
Reproduction step:
get in the left side bin, get ejected out instantly

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

Removes the useless bin from warden office. Closes #894
## Why It's Good For The Game

Doin your mom do do doin your mom
## Changelog
:cl:
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
everybody wanna feature code heliostation but nobody wanna bugfix selene

File: config/maps.txt [hunk 1]
```diff
@@ -20,6 +20,9 @@ map metastation
 	votable
 endmap
 
+map selenestation
```
Comment (Pepsilawn) [inline config/maps.txt:23]: hello

Comment (FernandoJ8) [reply config/maps.txt]: goodbye

Comment (OneAsianTortoise) [reply config/maps.txt]: I wanna say hello too

**Comment by remove32** [comment]
is simply removing the bin the right course of action here? how far is it to the nearest bin?

**Comment by SgtHunk** [comment]
> is simply removing the bin the right course of action here? how far is it to the nearest bin?

there's already a bin in the room i just had tunnel vision and put a 2nd one because i thought i didn't

**Pull request merged**